### PR TITLE
Add functionality to clear color from open tabs

### PR DESCRIPTION
**Pull request by kobilee**
I added a function to tackle this request in the todo list:
new command: clear the color of all open tabs
please review my code and let me know what you think

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -157,6 +157,28 @@ function unsetColor(context, title) {
...
 
+async function clearAllTabColors(context) {
+  const storage = new Storage(context);
+  const tabs = storage.get("tabs") || {};
+  const editors = vscode.workspace.textDocuments;
+  for (const editor of editors) {
+      const resource = editor.uri;
+      const fileName = resource.fsPath.replace(/\\/g, "\\\\");
+      for (const color in tabs) {
+          const tabIndex = tabs[color].indexOf(fileName);
```
Comment (mondersky) [inline extension.js:168]: use includes rather than indexOf for more readability

File: extension.js [hunk 2]
```diff
@@ -157,6 +157,28 @@ function unsetColor(context, title) {
   }
 }
 
+async function clearAllTabColors(context) {
```
Comment (mondersky) [inline extension.js:160]: it's better to name it clearOpenTabsColors to illustrate what it does more precisely

File: extension.js [hunk 3]
```diff
@@ -388,6 +410,9 @@ function activate(context) {
     storage.set("patchedBefore", false);
   });
 
+  disposable = context.subscriptions.push(vscode.commands.registerCommand('tabscolor.clearAllColors', () => {
```
Comment (mondersky) [inline extension.js:413]: same here, rename to clearOpenTabsColors

File: extension.js (hunk 2 above)
Comment (mondersky) [reply extension.js:160]: the function doesn't seem to do its job correctly, it looks like closed tabs are affected as well. Try doing the following:
- open multiple tabs and colors them all
- close one of them
- execute the command
- reopen the previously closed tab
You'll notice that the tab has lost its color even if it was not part of the open tabs
the function should only target open tabs

**Review by mondersky — CHANGES_REQUESTED** [review]
thanks for your contribution, I read your code and it looks 99% good to me so nice job !
but before merge, here are some points that need to be addressed

File: extension.js (hunk 2 above)
Comment (kobilee) [reply extension.js:160]: The functionality seems to be working as expected as you can see in the gif below, you already have a similar command to create all the colors. Were you executing that one by chance? if not please show me how you encountered the bug so I can recreate it.

[Example GIF](https://gifyu.com/image/SdCFb)

Comment (mondersky) [reply extension.js:160]: you're right, I was executing the other command thinking it was yours, in fact your function is working all fine.

Comment (kobilee) [reply extension.js:160]: No worries :) I will update the PR to have a better naming convention for the function!

File: extension.js (hunk 1 above)
Comment (kobilee) [reply extension.js]: Function updated and tested to use includes instead.

File: extension.js (hunk 3 above)
Comment (kobilee) [reply extension.js:411]: Function name updated

**Comment by mondersky** [comment]
great job !

**Pull request merged**